packages/vite/src/node/plugins/worker.ts

```diff
@@ -51,25 +51,46 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
       }
 
       let url: string
+      const isNode = (config.build.target as string).includes('node')
```

Comment:
**question:** Why is it safe to assume `string` here?

Reply:
@Shinigami92 I realized that there is a better way of doing this. Added isTargetNode utility function. Please have a look.